sql/moz-fx-data-marketing-prod/ga_derived/firefox_whatsnew_summary_v1/schema.yaml

```
@@ -0,0 +1,19 @@
+fields:
+- mode: NULLABLE
+  name: date
+  type: DATE
+- mode: NULLABLE
+  name: country
+  type: STRING
+- mode: NULLABLE
+  name: locale
+  type: STRING
+- mode: NULLABLE
+  name: version
+  type: STRING
```

And a corresponding schema change.
```suggestion
- mode: NULLABLE
  name: version
  type: STRING
- mode: NULLABLE
  name: major_version
  type: NUMERIC
- mode: NULLABLE
  name: minor_version
  type: NUMERIC
- mode: NULLABLE
  name: patch_revision
  type: NUMERIC
- mode: NULLABLE
  name: is_major_release
  type: BOOLEAN
```